index.Rmd

```diff
@@ -1,105 +1,594 @@
+adulttestingdata$Bmicpos=round((adulttestingdata$`B. microti (%)`/100)*adulttestingdata$Total_Tested)
+adulttestingdata$Bmiyapos=round((adulttestingdata$`B. miyamotoi (%)`/100)*adulttestingdata$Total_Tested)
+```
+```{r nymphzeroinfsetup,results='hide',message=FALSE, echo=F}
+nymphtestingdata$ANApos=round((nymphtestingdata$`A. phagocytophilum (%)`/100)*nymphtestingdata$Total_Tested)
+nymphtestingdata$Bmicpos=round((nymphtestingdata$`B. microti (%)`/100)*nymphtestingdata$Total_Tested)
+nymphtestingdata$Bmiyapos=round((nymphtestingdata$`B. miyamotoi (%)`/100)*nymphtestingdata$Total_Tested)
+```
+
+To examine the relationship between percent mixed/deciduous forest cover and <i> A. phagocytophilum </i>, <i> B. microti </i>, and <i> B. miyamotoi </i> prevalence, zero-inflated negative binomial (ZINB) models were used. The dependent variable for each count and zero-inflated model was the count of a particular pathogen, where the independent variable was percent mixed/deciduous forest cover. Code to create each model for both adults and nymphs is shown below:
+```{r adultregression,results='hide',message=F,error=F,warning=F}
+adult_ANA_reg=zeroinfl(formula = ANApos ~ Percent_Forest_Cover |  Percent_Forest_Cover,
+                 data=adulttestingdata,dist='negbin')
+adult_Bmic_reg=zeroinfl(formula = Bmicpos ~ Percent_Forest_Cover |  Percent_Forest_Cover,
+                 data=adulttestingdata,dist='negbin')
+adult_Bmiya_reg=zeroinfl(formula = Bmiyapos ~ Percent_Forest_Cover |  Percent_Forest_Cover,
+                 data=adulttestingdata,dist='negbin')
+```
+
+```{r nymphregression,results='hide',message=F,error=F,warning=F,comment=F}
+nymph_ANA_reg=zeroinfl(formula = ANApos ~ Percent_Forest_Cover |  Percent_Forest_Cover,
+                 data=nymphtestingdata,dist='negbin')
+nymph_Bmic_reg=zeroinfl(formula = Bmicpos ~ Percent_Forest_Cover |  Percent_Forest_Cover,
+                 data=nymphtestingdata,dist='negbin')
+nymph_Bmiya_reg=zeroinfl(formula = Bmiyapos ~ Percent_Forest_Cover |  Percent_Forest_Cover,
+                 data=nymphtestingdata,dist='negbin')
+```
+
+<br>
+
+Significance for adult ZINB models are shown below:
+```{r adulttable,echo=F,message=F,warning=F,comment=F}
+tab_model(adult_ANA_reg,adult_Bmic_reg,adult_Bmiya_reg,title = "Adult Regression Results",dv.labels = c("A. phagocytophilum","B. microti","B. miyamotoi"),show.r2 = F,show.ci=F,show.est = F,digits=2)
+```
+
+<br>
+
+Coefficient of adult <i> B. miyamotoi </i> model shown below:
+```{r adult_sig_coeff}
+adult_Bmiya_reg$coefficients$count
+```
+
+<br>
+
+Significance for nymphal ZINB models are shown below:
+```{r nymphtable,echo=F,message=F,warning=F,comment=F}
+tab_model(nymph_ANA_reg,nymph_Bmic_reg,nymph_Bmiya_reg,title = "Nymphal Regression Results",dv.labels = c("A. phagocytophilum","B. microti","B. miyamotoi"),show.r2 = F,show.ci = F,show.est = F,digits=2)
+```
+
+<br>
+
+Coefficient of statistically significant nymphal models shown below:
```

Comment:
not required, but check out functions in `library(broom)` for summarizing various models.